jinja2_standalone_compiler/__init__.py

```diff
@@ -4,14 +4,23 @@
 import os
 import re
 import sys
-
 import click
+from colorama import init, Fore, Style
```

Comment:
It's a good/common practice when using colorama (1) no to include colorama in `requirements.txt` and (2) make lib work with or without it (on a somewhat compatilibity mode). Since colorama is definitely not needed for `jinja_standalone_compiler` to work I guess we should use this approach as well.